components/MemoryGrid/MemoryGrid.styled.js

```diff
@@ -27,14 +27,15 @@ export const GridImageFront = styled.div`
   width: 100%;
   height: 100%;
   position: absolute;
-  backface-visibility: hidden;
+  /*   backface-visibility: hidden; */
```

Comment:
i guess this is for testing? remember to add this back in before merging…

Reply:
uff forgot that 😇